repository/config/grid.pan

```diff
@@ -22,6 +22,11 @@ variable REPOSITORY_GRID_THIRD_PARTY_VARIANT ?= 'emi';
 variable REPOSITORY_GRID_PREFIX ?= REPOSITORY_GRID_VARIANT + '_3_0_' + OS_VERSION_PARAMS['major'] + '_' + PKG_ARCH_DEFAULT;
 variable REPOSITORY_GRID_THIRD_PARTY_PREFIX ?= REPOSITORY_GRID_THIRD_PARTY_VARIANT + '_3_0_' + OS_VERSION_PARAMS['major'] + '_' + PKG_ARCH_DEFAULT;
 
+@{
+desc = defines the repository to use for WLCG RPMs.
+values = any string
+}

```

Comment:
Would be good to add `default` and `required` as for other variables. for the default value, put a text explaining how it is defined as it is not a constant value.